porfolio.html

```diff
@@ -50,6 +50,199 @@ <h1 class="primary-text">Hey There. <br>
     </section>
     
      </div>
+    <div class="my-recent-works">
+        <header class="recent-works">
+            <h2 class="header-content">
+                My Recent Works
+            </h2>
+            <hr class="hline">
+        </header>
+        <div class="recent-work-container">
+
+            <section class="featured-image">
+               <img src="./images/yoga.png" alt="a place holder image" class="yoga"> 
+                <div class="right-block">
+                    <h3 class="title-post">Multi-Post Stories</h3>
+                    <p class="supporting-text-rw">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.  has been the industry's 
+                        standard dummy text ever since the 1500s,
+                         when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="tags">
+                        <li class="css"><a href=""><small class="csst">css</small></a></li>
+                        <li class="html"><a href="" ><small class="htmlt">html</small></a></li>
+                        <li class="bootstrap"><a href="" ><small class="bootstrapt">bootstrap</small></a></li>
+                        <li class="ruby"><a href="" ><small class="rubyt">Ruby</small></a></li>
+                    </ul>
+                    <button type="button" disabled class="see-project"><small class="see-projectt">See Project</small></button>
+
+                </div>    
+           
+            </section>
+
+            <div class="card-work">
+
+                <section class=" cardworks cardwork-one">
+                    <div class="action">
+                        
+                        <button type="button" disabled class="btn"><small class="btntxt">See Project</small></button>
+                        <hr class="rectangle">
+                    </div>
+                    <article class="cardwork-one-rb">
+                        <p class="cwo-title-post">Profesional Art <br>Printing Data</p>
+                        <p class="cwo-supporting-text">A daily selection of privately <br>
+                            personalized reads; no accounts or <br>
+                            sign-ups required. has been the <br>
+                            industry's standard</p>
+                        <ul class="cwo-tags">
+                            <li class="cw-li cw-html"><a href="" ><small class="cw-text">html</small></a></li>
+                            <li class="cw-li cw-bootstrap"><a href="" ><small class="cw-text">bootstrap</small></a></li>
+                            <li class="cw-li cw-ruby"><a href="" ><small class="cw-text">Ruby</small></a></li>
+                        </ul>
```

Comment:
- [ ] Please stick to the design as much as possible. please remove the underline from `li` tag so it matches the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/213196767-c1e30eda-73aa-42c1-9423-a6c29a776cf0.jpg)